### PR TITLE
Updated broken ut47.h maxtrix

### PR DESCRIPTION
**Pull request by Valgrahf**
Added some keymaps and keymap changed as well.

### CODE REVIEW

**Comment by Valgrahf** [comment]
Updated corrupt merge on keyboards/ut47/keymaps/default/keymap.c

File: .wget-hsts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# HSTS 1.0 Known Hosts database for GNU Wget.
```
Comment (drashna) [inline .wget-hsts]: Could you remove this file?

File: keyboards/ut47/ut47.h [hunk 2]
```diff
@@ -22,26 +22,26 @@
...
   K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, \
-  K30, K31, K32, K33, K34,    K35,   K37, K38, K39, K3a, K3b  \
+  K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b  \
 ) \
 	{ \
     { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b }, \
     { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b }, \
     { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b }, \
-    { K30, K31, K32, K33, K34,    K35,   K37, K38, K39, K3a, K3b }  \
+    { K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3a, K3b }  \
```
Comment (drashna) [inline keyboards/ut47/ut47.h]: Is the K36 needed here? 

@filoxo appears to have fixed this in another PR, and since it looks like he's the creator...

Comment (Valgrahf) [reply keyboards/ut47/ut47.h]: Ah, he beat me to it.

**Comment by drashna** [comment]
Looks like you deleted the file altogether!

**Comment by drashna** [comment]
@Valgrahf If you could, add this: 

```c
#define LAYOUT( \
  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b, \
  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b, \
  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b, \
  K30, K31, K32, K33, K34,    K35,   K37, K38, K39, K3a, K3b  \
) \
  { \
    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0a, K0b }, \
    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1a, K1b }, \
    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2a, K2b }, \
    { K30, K31, K32, K33, K34, K35, KC_NO, K37, K38, K39, K3a, K3b }  \
  }
```

The K35 is in the wrong spot.  (#2808 )

**Pull request closed without merging**